src/ts/components/settingstogglebutton.ts

```diff
@@ -41,7 +41,20 @@ export class SettingsToggleButton extends ToggleButton<SettingsToggleButtonConfi
       text: i18n.getLocalizer('settings'),
       settingsPanel: null,
       autoHideWhenNoActiveSettings: true,
+      role: 'pop-up button',
+      ariaLabel: i18n.getLocalizer('settings'),
     }, <SettingsToggleButtonConfig>this.config);
+
+    /**
+     * WCAG20 standard defines which popup menu (element id) is owned by the button
+     */
+    this.getDomElement().attr('aria-owns', config.settingsPanel.getActivePage().getConfig().id);
```

Comment:
Is WCAG20 really supposed to work with the ID of an element? What if we have multiple UI instances?

Reply:
https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships#aria-labelledby

All aria relationships work on element id's.

Aren't the id's always dynamically generated and unique for each instance of the UI?

Reply:
You are right, they are (usually).